src/EventTickets/resources/templates/EventTickets/EventTicketsListHOC.tsx

```diff
@@ -36,15 +36,15 @@ export default function EventTicketsListHOC({name, tickets, ticketsLabel, soldOu
         }
 
         setValue(name, JSON.stringify(Object.values(selectedTickets)));
-    }, [tickets, selectedTickets]);
+    }, [ticketTypes, selectedTickets]);
 
-    const onSelectTicket: OnSelectTicketProps = (ticketId, ticketQuantity, ticketPrice) => (selectedQuantity) => {
+    const onSelectTicket: OnSelectTicketProps = (ticketId, totalTickets, ticketPrice) => (selectedQuantity) => {
         if (selectedQuantity < 0) {
             selectedQuantity = 0;
         }
 
-        if (selectedQuantity > ticketQuantity) {
-            selectedQuantity = ticketQuantity;
+        if (selectedQuantity > totalTickets) {
```

Comment:
Does `totalTickets` mean the total tickets allowed or the total tickets remaining?